packages/demo-saas/src/TableExample.tsx

```diff
@@ -1149,7 +1149,7 @@ export const TableExample: FC = () => {
   }, []);
 
   return (
-    <div style={{ textAlign: 'center' }}>
+    <div style={{ textAlign: 'center', paddingBottom: '20px' }}>
```

Comment:
why exactly 20px? it this pagination height?

Reply:
it's just padding for div container in the example, it's not related to the table